asserts/database.go

```diff
@@ -30,22 +30,19 @@ import (
 )
 
 // Backstore is a backstore for assertions. It can store and retrieve
```

Comment:
Can this be `Backstore for assertions`? or is there a convention about having a verb there that we must follow?